java/src/org/dartlang/vm/service/VmServiceBase.java

```diff
@@ -190,6 +193,18 @@ public static VmService localConnect(int port) throws IOException {
   RequestSink requestSink;
 
   /**
+   * The [addBreakpoint] RPC is used to add a breakpoint at a specific line of some script. This
+   * call works for protocol versions 2.1 and 3.0
```

Comment:
@turnidge @johnmccutchan @rmacnak-google, fyi, it looks like the spec version in the markdown file and the one reported by the VM has been out of sync at times.